### PR TITLE
Moff cooking. Part one.

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

Add baking for moth.
Cotton dough = 5 flour + 10 water + 10 fabric
Bread = Cotton dough + 10 sec microwave
Pizza = Flat cotton dough + 4 cotton bolls + 30 sec microwave

## Why / Balance

Moff rejoice!

## Technical details

no

## Media

https://github.com/user-attachments/assets/a09d5644-abb0-4a84-b475-5eeaa8c793d0

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Cotton dough added to the game! Check the guidebook for new recipes.

### CODE REVIEW

**Comment by Hrosts** [comment]
Could alternatively make it from fiber as a sort of mothy flour replacer, and add fiber bags to the chef's vendor/lockers.

**Comment by lzk228** [comment]
> Could alternatively make it from fiber as a sort of mothy flour replacer, and add fiber bags to the chef's vendor/lockers.

nah, chef can butch some clothing or request cotton from botany

**Comment by LankLTE** [comment]
This is nice. I like this.

**Comment by Minty642** [comment]
if it possible, can you make it flammable?

**Comment by lzk228** [comment]
possible, but i don't want it

File: Resources/ServerInfo/Guidebook/Service/FoodRecipes.xml [hunk 1]
```diff
@@ -22,9 +22,15 @@ WARNING: This is not an automatically generated list, things here may become out
 - Chocolate = 6 Cocoa Powder, 2 Milk, 2 Sugar
 - Uncooked Animal Protein: Grind Raw Meat
 
+Buzz! Don't forget about Moth diet!
+- Cotton Dough = 5 Flour, 10 Fabric, 10 Water
+- Almost every other basic recipe also works with cotton dough (e.g. bread)
```
Comment (UbaserB) [inline Resources/ServerInfo/Guidebook/Service/FoodRecipes.xml:27]: this is not true since there's only recipes, either change or fix that please

**Review by UbaserB — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/Guidebook/Service/FoodRecipes.xml (hunk 1 above)
Comment (lzk228) [reply Resources/ServerInfo/Guidebook/Service/FoodRecipes.xml:27]: weh

**Review by UbaserB — APPROVED** [review]

**Comment by lzk228** [comment]
moth rejoice!

**Pull request merged**